### ISSUE TITLE
Add Description tooltip position(relative and absolute)

### ISSUE THREAD

**Issue by GENL** [SimformSolutionsPvtLtd/flutter_showcaseview#54]
The widget will be far better advanced if there's the possibility to set the tooltip the place we want around the widget that `Showcase` wraps. And may be the same with the arrow.

**Comment by jlund24** [comment]
Yes please. It's basically impossible to showcase items that go from the top to bottom of the screen (such as a drawer) and I'm having very poor luck getting the titles and descriptions to show up above widgets

**Comment by GENL** [comment]
Take a look at this library that does what I mention https://pub.dev/packages/bubble_showcase

**Comment by colbymaloy** [comment]
this should be a top priority. seems like a major oversight not to have left and right alignments for this
